### DISCUSSION TITLE
Documentation addition: Google as an OIDC

### DISCUSSION THREAD

**Discussion by initstring** [SecurityRiskAdvisors/VECTR#165] [General]
Hi team!

Are you interested in documentation suggestions? I went through configuring Google as an identity provider today. It works great, but having the following in the Vectr documentation would have made the process much easier.

Perhaps it would make sense to add a section for common providers and known issues/values?

Here is what was key:

Type: OIDC

**Identity Provider**:
Discovery URL: `https://accounts.google.com/.well-known/openid-configuration`

**Claims Mapping**:
username: `email` (default value does not work and will generate broken username entries)

Once again, thanks for a great product!!!

**Comment by doodleincode** [comment]
Hey initstring!

Thanks for the great feedback and sharing your configuration for integrating Google as an OIDC provider.

For now, I think it makes sense to have community driven documentation shared on Discord or here on Github. This is especially the case for SSO as we do not provide configuration support for this feature. I can move this over to Github Discussions and link it to a post on Discord if that would be helpful.

**Comment by initstring** [comment]
Thanks @doodleincode!

That sounds perfect. I wasn't aware of GH Discussions being used, I will share things like this there next time.

Have a great day!

Edit to add: If I'm able to edit the content when it is moved to discussions, I can add more details to make it a more thorough overview as well.
